models/User.js

```diff
@@ -0,0 +1,8 @@
+
+class User {
```

Comment:
you need just to export class in es6 way
export class User {
	constructor(){.....